ui/src/reusable_ui/components/wizard/WizardStep.tsx

```diff
@@ -50,22 +62,34 @@ class WizardStep extends PureComponent<Props> {
 
   private handleClickPrevious = async () => {
     const {onPrevious, decrement} = this.props
+    let returnedValue
     if (onPrevious) {
-      await onPrevious()
+      returnedValue = await onPrevious()
     }
     if (decrement) {
       decrement()
     }
+    return returnedValue
   }
 
   private handleClickNext = async () => {
-    const {onNext, increment} = this.props
+    const {onNext, increment, isBlockingStep} = this.props
+    let payload
+    let status = true
```

Comment:
This name of status feels a little awkward and the variable looks like it can hold a number or a boolean which seems a little weird.